modules/ROOT/pages/cache.adoc

```diff
@@ -0,0 +1,25 @@
+= Optionally Configure the Cache
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint API Community Manager can improve performance with a platform cache.
+
+The platform cache has two parts: a session cache that holds data specific to one user and their session, and an organization cache that holds data that will be accessed by multiple users in the organization.
+
+If you wish to change the default configuration for the caches:
+
+. Log in to your Salesforce organization.
+. In *Setup*, click *Custom Metadata Types*.
+. Next to the label *ACM Platform Cache Metadata*, click *Manage Records*.
+. Next to the label *ACM Configuration*, click *Edit*.
+. Use the *OrgCacheEnabled* and *SessionCacheEnabled* checkboxes to enable or disable the caches.
+. Set the number of seconds that cached data remains in the cache for the API documentation, API instance, and API query entities with the *ApiDocumentationLifetime*, *ApiInstancesLifetime*, and *ApiQueryLifetime* fields.
+. Hover the mouse cursor over the question mark next to each field name to see the minimum and maximum number of seconds for each entity.
+. Set a field to zero seconds to disable caching of that entity.
+
```

Comment:
Click SAVE to save the changes. The new configuration becomes effective immediately.

Reply:
ADD: 
- click save to save the changes.

Caches are by default configured to 0 size. To complete configuration of the caches, you need to set the size to more than 0. To configure the size of the caches, see current usage and clear their entries:
1. In *Setup*, go to "Platform Cache"
2. Open the `ACMCache` partition
3. Click Edit, and define a size for each cache using the "Organization" field. Observe the sum of the sizes of both caches must be less than or equal to the cache capacity in your org. [LINK TO Licensing overview section]
4. Click save

The new configuration becomes effective immediately. In this section you will be able to see how caches are being used, by clicking on Recalculate button.

Reply:
OBSERVE these sizes are in Mb

Reply:
Objects that are cached include:

```
Object             Description 
Applications               Details of Client Applications of users
API Versions               Basic details of APIs versions from Exchange
API Documentation    API version information and documentation pages
API Instances              Available instances of each published API version
API Query                    Responses of the TRY IT operation in the API Console
```

Reply:
Added all of this. Thank you!